### ISSUE TITLE
Suggestion: Split Framework and CMS into two distinct projects

### ISSUE THREAD

**Issue by arsenikstiger** [OrchardCMS/OrchardCore#3143]
Hi,

I would suggest to split Framework and CMS into two distinct projects, to clarify code, issues and documentation.
Today, all these things are merged into one global project, and make it unreadable.
I only want to use the framework part, but it's difficult to find some documentation and issues regarding my problems, because all usages are joining here.

Regards

**Comment by infofromca** [comment]
not just for  documentation and issues, more important is project (or even code ) portable.
for example, I only need a simple site and Email

**Comment by sebastienros** [comment]
I am not opposed to it. Even more since the framework is getting more stable and fewer changes are done on it.

**Comment by sebastienros** [comment]
Today everything is already modular, it won't make application be easier to understand. The only reason to do it is 
- Marketing as people might confuse the framework and the CMS. Having two repos might help.
- Technical as devs might find it easier to have smaller solutions. This could be done in the single repos.
- Other CMSes would be more inclined to use the framework if it was not linked to the "Orchard" term which wrongly means _CMS_ for many dotnet devs.

**Comment by matiasmolleja** [comment]
They seem pretty good reasons to me.
OrchardCore as the basis for : OC-CMS, OC-ECommerce, OC-CRM, OC-DAM, OC-WorkFlowManager...

**Comment by arsenikstiger** [comment]
I'm happy to see my vision is shared !
What I wanted to say is the CMS part is itself an implementation of OrchardCore Framework.
Framework is interesting for pure developers where CMS can be interesting for both developers and endusers.
To finish, CMSes are linked to blog and marketing sites, where Orchard can be so more than only that !
It can be a framework on which commercial societies can develop their business modules.
PHP or java are saturated, but there is a big place left for a challenger in .NET.

**Comment by china-live** [comment]
我们需要的仅仅是一个模块化的开发框架和一个简单的管理后台 ，
并不需要内容管理系统。
当前，内容管理系统已经过时了;
网站前端直接用Razor Pages来做更实在，灵活性更高，学习成本更低。

All we need is a modular development framework and a simple admin system.
No content management system is required.
Currently, content management systems are outdated;
Web front-ends are more straightforward to develop with Razor Pages, with greater flexibility and lower learning costs.

**Comment by china-live** [comment]
It is silly to let the designer or the person who does not understand the code develop the website itself, especially the current user requirements for the UI are extremely high.

**Comment by china-live** [comment]
我的意思内容管理系统只是对发布的内容进行管理，网站设计和排版不应该属于CMS范畴内的东西，现在的CMS Modules 集成了太多与显示相关的东西，造成整个项目臃肿，开发时间长，学习成本高。
I mean the content management system only manages the published content. Website design and layout should not belong to the CMS category. Now CMS Modules integrates too many things related to display, resulting in bloated projects and long development time. The cost of learning is high.

想一想，OrchardCore项目到目前为止开发多长时间了，照这样下去，没等开发完成就已经过时了
Think about how long the Orchard Core project has been developed so far. If you continue this way, you will be out of date without waiting for the development to complete.

I wait for the flowers have withered

**Comment by scleaver** [comment]
I don’t agree - pure “headless” CMS’s are not responsible for display and layout but OrchardCore is not a headless only CMS it is a WCMS as well - capable of both headless and traditional web content management. There is still a huge place for WCMS so rightly Orchard does both.

**Comment by mmdqq** [comment]
So i think ,OrchardCore can be a platform，not only a CMS ,It is betweent  cloud and client,in fact ,i am developing a system for gov base on OrchardCore ,user can use many different cloud feature in one app

**Comment by agriffard** [comment]
See https://twitter.com/dodyg/status/1321893431376072704
